src/earthdataVarinfo/handler.py

```diff
@@ -0,0 +1,62 @@
+"""Module providing access to environment variables"""
+import os
+from tempfile import TemporaryDirectory
+
+from varinfo import VarInfoFromNetCDF4
+from varinfo.cmr_search import get_granules, get_granule_link, download_granule
+from varinfo.umm_var import get_all_umm_var
+
+def main(event, context):
+    """Handler that calls the earthdata-varinfo library"""
+
+    # Setup CMR search url from cmrRootUrl
+    cmr_url = os.environ['cmrRootUrl'] + '/search/'
```

Comment:
Where are you getting `cmrRootUrl` from? Do you think it would be better if you imported CMR like:

```from cmr import CMR_UAT```

That's what we've been doing with `earthdata-varinfo` and would be good for consistency

Reply:
cmrRootUrl is an environment variable in serverless.yml. That is where we store environment variables in cmr-graphQL. I believe its best practice to get these URLs from environment variables.